packages/mobile/test/schemas.ts

```diff
@@ -63,6 +63,7 @@ export const vNeg1Schema = {
   geth: {
     initialized: 'INITIALIZED',
     connected: true,
+    promptZeroSyncIfNeeded: false,
```

Comment:
state schema changes, including new additions, need to go in the latest schema (v5 below) since it didn't exist in v-1